code/modules/mob/living/carbon/human/human_mob.dm

```diff
@@ -1931,7 +1931,7 @@ Eyes need to have significantly high darksight to shine unless the mob has the X
 /mob/living/carbon/human/get_runechat_color()
 	if(client?.prefs.toggles2 & PREFTOGGLE_2_FORCE_WHITE_RUNECHAT)
 		return "#FFFFFF" // Force white if they want it
-	return dna.species.get_species_runechat_color(src)
+	return runechat_color
```

Comment:
The above toggle is also kinda redundant now that its just a blanket colour option

Reply:
We still have it in the case that they don't want their runechat to be determined by a specific color, if we were to remove it we'd have to update the SQL among other things. This also just allows people to toggle it at a whim, which I am all for.

Reply:
It wouldnt need an SQL update if its just a pref toggle were not needing.

The force white toggle isn't needed if they can just manually set their colour to `#ffffff` in settings.

Reply:
That is fair, I'll remove it since it's no longer needed.

Reply:
> It wouldnt need an SQL update if its just a pref toggle were not needing.
> 
> The force white toggle isn't needed if they can just manually set their colour to `#ffffff` in settings.

Alright, fully done. `PREFTOGGLE_2_FORCE_WHITE_RUNECHAT` is no more.